### ISSUE TITLE
Merge conflicts

### ISSUE THREAD

**Issue by kaz7** [sx-aurora-dev/llvm-project#34]
@simoll -san, is it possible to check your modifications something like below on your merge-conflict-resolve in cc48023, 84c3fca, f64cf08, 810eb60, 918e046, and fb109a7?
```
@@@ -164,7 -165,7 +165,7 @@@ void BranchFolder::RemoveDeadBlock(Mach

    // Update call site info.
    std::for_each(MBB->begin(), MBB->end(), [MF](const MachineInstr &MI) {
 -    if (MI.shouldUpdateCallSiteInfo())
 +    if (MI.isCandidateForCallSiteEntry())
        MF->eraseCallSiteInfo(&MI);
    });
    // Remove the block.
```
You modify shouldUpdateCallSiteInfo to isCandidateForCallSiteEntry each time as a part of resolve for merge conflicts.  However, 016d91c modifies it to use shouldUpdateCallSiteInfo in upstream.

Did you perform this without intension?  If so, I'll take upstream's code.  Otherwise, please let me know.

**Comment by simoll** [comment]
This was unintentional. Upstream is the way to go here.

**Comment by kaz7** [comment]
Thanks!

**Comment by kaz7** [comment]
I've fixed this by c481a57.  I'm going to close this ticket.
